### PR TITLE
Update tools list

### PR DESCRIPTION
**Pull request by AnnaNodes**
Hello,

We are a Casper node provider NOWNodes. 

Could you update the list of developer tools?

### Related links

// Paste links to Github tasks or links to related PRs.

### Changes

// Paste brief description what has changed.

// Paste [loom](https://www.loom.com/), screenshots, or gifs([record](https://giphy.com/apps/giphycapture), [compress](https://gifcompressor.com/)) here.

### Notes

// Paste any other notes if any

### CODE REVIEW

**Comment by ipopescu** [comment]
@piotr-dziubecki, could the Casper Association take a look at this, please?

**Comment by ipopescu** [comment]
@AnnaNodes, we do not list validators on this page.
Please work with https://casperecosystem.io/#ecosystem; press the button "Add project".  Make sure you have Casper-specific documentation and tools before adding the project (similar to [this](https://documenter.getpostman.com/view/13630829/TVmFkLwy#0c530825-c261-488a-9792-722b6107322e)).

**Comment by AnnaNodes** [comment]
Hello @ipopescu !

We do not provide validators. We provide full nodes. 
Could you list our project? Casper Association has approved the listing.
Снимок экрана 2023-01-09 в 12 23 58

**Comment by ipopescu** [comment]
@AnnaNodes, so these are read-only nodes, then? If so, we can probably list this URL that is specific to Casper: https://nownodes.io/nodes/casper-cspr. Please include me in the conversation (Telegram:**@iulia_telegram** or iulia@casperlabs.io) to finalize this.

**Comment by AnnaNodes** [comment]
I will gladly add you in the telegram chat. 
Could you add me to your list of contacts for that? (https://t.me/AnnaIts)

File: source/docs/casper/dapp-dev-guide/build-on-casper/index.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ The Casper Ecosystem is growing every day through the addition of new dApps and
 
 ### Developer Tools
 - [casperholders.io](https://casperholders.io)
+- [NOWNodes.io](http://bit.do/github-casper)
```
Comment (ipopescu) [inline source/docs/casper/dapp-dev-guide/build-on-casper/index.md:32]:
```suggestion
- [NOWNodes.io](https://nownodes.io/nodes/casper-cspr)
```

Comment (ipopescu) [reply source/docs/casper/dapp-dev-guide/build-on-casper/index.md:32]: The original link doesn't seem to work for me. Can we use this link instead? If so, I can approve and merge this PR.

**Review by ipopescu — APPROVED** [review]

**Pull request merged**